corpus/convert.py

```diff
@@ -109,29 +112,42 @@ class CorpusToStmJob(Job):
     Convert a Bliss corpus into a .stm file
     """
 
+    __sis_hash_exclude__ = {"non_speech_tokens": None, "punctuation_tokens": None}
+
     def __init__(
         self,
-        bliss_corpus,
-        exclude_non_speech=True,
-        remove_punctuation=True,
-        fix_whitespace=True,
-        name="",
-        tag_mapping=(),
+        bliss_corpus: Path,
```

Comment:
```suggestion
        bliss_corpus: Path,
        *,
```

Reply:
As you added new parameters that are not at the end, you should force the user to use named parameters (for all parameters except `bliss_corpus`), to not have accidentally wrong parameter assignment.